dask/dataframe/core.py

```diff
@@ -132,21 +133,64 @@ def finalize(results):
 class Scalar(DaskMethodsMixin, OperatorMethodMixin):
     """A Dask object to represent a pandas scalar"""
 
-    def __init__(self, dsk, name, meta, divisions=None):
+    def __init__(self, dsk=None, name=None, meta=None, divisions=None, operation=None):
```

Comment:
Not sure of the best way to support "old" and "new" mechanisms for initializing collections.  Other ideas are welcome.

The vision here is to make the entire state of a collection depend on the `operation`. However, we are a pretty long way from being able to **require** collections to be intialized this way.  Also, when we **do** remove the legacy arguments, it may make sense to support the `data=` behavior of pandas (and automatically create a creation operation under the hood).